### PR TITLE
bugfix: stray error 'error: expected declaration specifiers or ‘...’ …

### PR DESCRIPTION
**Pull request by mrigendrachaubey**
…before string constant'

Signed-off-by: Mrigendra Chaubey <Mrigendra.Chaubey@PathPartnertech.com>

### CODE REVIEW

**Comment by mrigendrachaubey** [comment]
make build fails on 4.15.0-47-generic ubuntu16.04 LTS.

File: tools/labs/templates/kernel_modules/3-error-mod/err_mod.c [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #include <linux/init.h>
 #include <linux/kernel.h>
-/* TODO: add missing kernel headers */
 #include <linux/module.h>
```
Comment (dbaluta) [inline tools/labs/templates/kernel_modules/3-error-mod/err_mod.c]: Are you sure this is the correct fix? We intentionally added the TODO there. This exercises shows the reader what happens if the proper headers are not included.

Thanks for the contribution! But I think you should rethink your solution. If there is a stray character before the comment just remove that but please leave the comment there.

**Comment by dbaluta** [comment]
> make build fails on 4.15.0-47-generic ubuntu16.04 LTS.

Yes, the build failure is as expected. There is an exercise which asks the students to fix the build failure.

@razvand @tavip Should we remove this exercise. It is confusing.

File: tools/labs/templates/kernel_modules/3-error-mod/err_mod.c (hunk 1 above)
Comment (mrigendrachaubey) [reply tools/labs/templates/kernel_modules/3-error-mod/err_mod.c]: Its a wrong fix definetly and I wasn't aware if it was an exercise, as I was also getting build failure in doing make docs :p

https://github.com/sphinx-doc/sphinx/issues/6269

As I am getting my senses back I see this is wrong solution I have posted.

Comment (mrigendrachaubey) [reply tools/labs/templates/kernel_modules/3-error-mod/err_mod.c]: Only thing that bothers me here is without using the comments I can compile it easily. So what headers we need, as error says nothing about header missing.

**Comment by razvand** [comment]
@dbaluta, I think it's a good exercise. It'a a troubleshooting exercise showing what happens when you don't include all headers. We should make it more ambiguous in the exercise statement, but I wouldn't remove it. Please add an issue and close this pull request.

**Comment by dbaluta** [comment]
Added the following issue to find a better solution: https://github.com/linux-kernel-labs/linux/issues/132

**Pull request closed without merging**